src/features/identityCheck/pages/identification/IdentityCheckValidation.tsx

```diff
@@ -28,7 +28,7 @@ export const IdentityCheckValidation = () => {
     <PageWithHeader
       title={t`Mon identité`}
       fixedTopChildren={
-        <CenteredTitle title={t`Les informations extraites sont-elles correctes ?`} />
+        <CenteredTitle title={t`Les informations extraites sont-elles correctes\u00a0?`} />
```

Comment:
je vois que finalement c'est l'hexadécimal qui a été choisi `\u00a0`

`\u00a0` n'est pas la chose la plus simple à lire et à retenir, j'ai l'impression que c'est une `magic string`

j'ai connu des expériences passés, où des collègues se sont plantés d'un tout petit chiffre dans le code hexadécimal ... ce qui me ne rassure pas

j'aimerais bien avoir un code explicite et qui diminue les risques d'erreurs

je propose d'extraire ce code dans une constante qui serait défini à un endroit et réutilisé partout

```suggestion
        <CenteredTitle title={t`Les informations extraites sont-elles correctes${NBSP}?`} />
```

qu'en penses-tu ?

Reply:
je pensais aussi que `NBSP` serait plus facile à comprendre et maintenir. Mais je vois 2 inconvénients qui font que je penche plus vers intégrer `\u00a0` directement dans le message:
- faut importer `NBSP` de partout, et ça rend le texte moins lisible.
- le message généré est moins facile à lire et je suis même pas sûr qu'il passe le build

![image](https://user-images.githubusercontent.com/10118284/146559178-8a72b55d-f44f-4988-a3af-a22bab628a61.png)

Reply:
Ouhh je l'avais pas vu venir la mauvaise génération de trad par lingui-js…

Si la magic string te fait peur @bruno-ebstein-pass-culture, je finis d'écrire une règle eslint avec autofix pour remplacer un ` ?` par `\u00a0?`

Reply:
bonne idée la règle ESLint